src/base/output.h

```diff
@@ -111,7 +107,23 @@ class CVC5_EXPORT Cvc5ostream
   std::ostream* getStreamPointer() const { return d_os; }
 
   template <class T>
-  Cvc5ostream& operator<<(T const& t) CVC5_EXPORT;
+  Cvc5ostream& operator<<(T const& t)
+  {
+    if (d_os != NULL)
+    {
+      if (d_firstColumn)
+      {
+        d_firstColumn = false;
+        long indent = d_os->iword(s_indentIosIndex);
```

Comment:
Not sure whether we want to do this in this PR, or do it at all: I've consolidated all these bits attached to an ostream in `options/io_utils.h`. I'd suggest to merge this `s_indentIosIndex` into this. (and maybe move it from options to util)

Reply:
Seems like a reasonable solution, but I'd prefer to do that in another PR.